packages/admin/cms-admin/src/dam/DataGrid/duplicatedFilenames/ManuallyHandleDuplicatedFilenamesDialog.tsx

```diff
@@ -78,44 +98,37 @@ export const ManuallyHandleDuplicatedFilenamesDialog: React.VoidFunctionComponen
                 </StyledList>
             </DialogContent>
             <DialogActions>
-                <Button variant="outlined" onClick={onSkip}>
-                    <FormattedMessage
-                        id="comet.dam.duplicateFilenameDialog.action.skip"
-                        defaultMessage="Skip {count, plural, one {file} other {files}}"
-                        values={{
-                            count: filenameData.length,
-                        }}
-                    />
-                </Button>
-                <Button variant="contained" color="warning" onClick={onReplace}>
-                    <FormattedMessage
-                        id="comet.dam.duplicateFilenameDialog.action.replaceFiles"
-                        defaultMessage="Replace {count, plural, one {file} other {files}}"
-                        values={{
-                            count: filenameData.length,
-                        }}
-                    />
-                    <Tooltip
-                        trigger="hover"
-                        title={
+                <Box display="flex" justifyContent="space-between" width="100%">
+                    <Button variant="text" color="secondary" onClick={onSkip} startIcon={<Forward />}>
+                        <FormattedMessage
+                            id="comet.dam.duplicateFilenameDialog.action.skip"
+                            defaultMessage="Skip {count, plural, one {file} other {files}}"
+                            values={{
+                                count: filenameData.length,
+                            }}
+                        />
+                    </Button>
+                    <Box>
+                        <Button variant="outlined" onClick={onReplace} startIcon={<Duplicate />} sx={{ marginRight: "10px" }}>
```

Comment:
@jamesricky is this what you meant regarding the dialog actions?

Reply:
Exactly 👍🏼